### ISSUE TITLE
Compatibility with Windows

### ISSUE THREAD

**Issue by blakemorgan** [drush-ops/drush-launcher#47]
Since *.phar files may need to be executed with the `php` command, you can't run the command from anywhere. To mitigate this, create a `drush.bat`  file in the same folder as `drush.phar` with the following lines: 

```batch
@echo off
php %~dp0\drush.phar
```

That will run the Drush Launcher from anywhere.

**Comment by hansfn** [comment]
Yes, please add this to the documentation/readme. That would make it much easier for people on Windows to use Drush Launcher. 

And, it is consistent with the Composer [recommendation/docs for Windows](https://getcomposer.org/doc/00-intro.md#manual-installation).

**Comment by weitzman** [comment]
Windows users can call `drush.php` instead of `drush`. I think they are equivalent, and Windows will be happier with the .php at the end. If I'm right, we still need to document this.

**Comment by hansfn** [comment]
I'm not sure I understand your comment, weitzman. The recommended approach for downloading Drupal Launcher is as a phar file - there is no `drush.php`. 

If your comment is about skipping Drush Launcher and calling Drush (directly) which might make sense if you only have one codebase on your system, then I can tell you that calling `drush.php` is very fragile. On my Windows machine it opens an editor. The only safe approach is creating bat-files.

Or maybe I misunderstood your comment completely.

**Comment by emb03** [comment]
If drush.php is fragile and we are to make a .bat file are there instructions anywhere on how to make one? This is the only command that worked for drush for me.

**Comment by hansfn** [comment]
Did you read [the installation instructions](https://github.com/drush-ops/drush-launcher#installation---phar) - the last item is about Windows? 

It's basically the same as the first post in this issue, so maybe I don't understand your question.
